src/client/Microsoft.Identity.Client/Platforms/netcore/NetCoreWebUIFactory.cs

```diff
@@ -26,12 +26,17 @@ internal class NetCoreWebUIFactory : IWebUIFactory
             if (webViewPreference == WebViewPreference.Embedded)
             {
                 throw new MsalClientException(MsalError.WebviewUnavailable,
-                    "If you have a Windows application which targets net5 or net5-windows, please change the target to net5-windows10.0.17763.0, " + 
-                        "which provides support from Win7 to Win10. For details, see https://github.com/dotnet/designs/blob/main/accepted/2020/platform-checks/platform-checks.md" +
-                    "If you have a cross-platform (Windows, Mac, Linux) application which targets net5, please dual target net5 and net5-windows10.0.17763.0." + 
-                        "Your installer should deploy the net5 version on Mac and Linux and the net5-window10.0.17763.0 on Win7 - Win10." + 
+                    "If you have a Windows application which targets net5.0 or net5.0-windows, change the target to net5.0-windows10.0.17763.0, " +
+                        "which provides support from Windows 7 to Windows 10. For details, see https://github.com/dotnet/designs/blob/main/accepted/2020/platform-checks/platform-checks.md" +
+
+                    "If you have a cross-platform (Windows, Mac, Linux) application which targets net5.0, dual target net5.0 and net5.0-windows10.0.17763.0." +
+                        "Your installer should deploy the net5.0 version on Mac and Linux and the net5.0-window10.0.17763.0 on Windows." +
                         "For details, see https://github.com/dotnet/designs/blob/main/accepted/2020/platform-checks/platform-checks.md" +
-                    "If you have a .NET Core 3.1 app, please reference the NuGet package Microsoft.Identity.Client.Desktop and call the extension method .WithDesktopFeatures() first." + "For details, see https://aka.ms/msal-net-webview2 or use the system WebView - see https://aka.ms/msal-net-os-browser");
+
+                    "If you have a .NET Core 3.1 app, reference the NuGet package Microsoft.Identity.Client.Desktop and call the extension method .WithDesktopFeatures() first." + "For details, see https://aka.ms/msal-net-webview2 or use the system WebView - see https://aka.ms/msal-net-os-browser" +
+
+                    "If your application targets .NET 6, use Windows broker by installing the NuGet package Microsoft.Identity.Client.Broker and calling the extension method .WithBrokerPreview() first. For details, see https://aka.ms/msal-net-wam ");
```

Comment:
@bgavrilMS One of the limitations here is net6 targets won't have embedded webview available. Maybe an edge in cases if someone can't use broker or OS browser.